cypress/e2e/1/delete-pizza.cy.js

```diff
@@ -0,0 +1,35 @@
+describe('Delete pizza', () => {
```

Comment:
why do we need all that example tests?

Reply:
This is used for local test validation of cypress plugin. It is not supposed to be included in the published distribution of the package.

Reply:
ok, but if we decide to keep tests then we need to setup ci/cd for it and also add tests for customisation